website/docs/reference/node-selection/methods.md

```diff
@@ -258,7 +258,7 @@ The `exposure` method is used to select parent resources of a specified [exposur
   ```bash
 dbt run --select "+exposure:weekly_kpis"                # run all models that feed into the weekly_kpis exposure
 dbt test --select "+exposure:*"                         # test all resources upstream of all exposures
-dbt ls --select "+exposure:*" --resource-type snowplow  # list all sources of type "snowplow" upstream of all exposures
+dbt ls --select "+exposure:*" --resource-type source    # list all source tables upstream of all exposures
```

Comment:
This lines up with the ["resource method" section](https://docs.getdbt.com/reference/node-selection/methods#the-resource_type-method) of this page:
>Note: This method doesn't work for sources, so use the [--resource-type](https://docs.getdbt.com/reference/commands/list) option of the list command instead:
> dbt list --resource-type source

And when I looked at the usage of the [dbt ls command](https://docs.getdbt.com/reference/commands/list):
> [--resource-type {model,semantic_model,source,seed,snapshot,metric,test,exposure,analysis,default,all}]